fiftyone/core/camera.py

```
@@ -0,0 +1,848 @@
+
+        import fiftyone as fo
+
+        intrinsics = fo.PinholeCameraIntrinsics(
+            fx=1000.0,
+            fy=1000.0,
+            cx=960.0,
+            cy=540.0,
+        )
+    """
+
+    pass
```

Ruff (0.14.10)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)